pep_schema.yaml

```diff
@@ -49,6 +50,7 @@ properties:
       required:
         - input_file_path
         - output_file_path
```

Comment:
should we rename `output_file_path`? 

it's weird to see:
- `output_file_path` 
- `output_bigbed_path`.

expect both or neither to include format.

Reply:
I agree. 

Also, I'm not sure why `(derived attribute)` has been included in the property description in both cases. Technically, it does not have to be derived